target/ppc/insn64.decode

```diff
@@ -23,6 +23,10 @@
 @PLS_D          ...... .. ... r:1 .. .................. \
                 ...... rt:5 ra:5 ................       \
                 &PLS_D si=%pls_si
+&PLF_D          frt ra si:int64_t r:bool
+@PLF_D          ...... .. . .. r:1 .. .................. \
+                ...... frt:5 ra:5 ................       \
+                &PLF_D si=%pls_si
```

Comment:
Can we reuse PLS_D instead? Then you could move `resolve_PLS_D` from `fixedpoint-impl.c.inc` to `translate.c` and avoid creating a `resolve_PLF_D`.